src/components/Contentful/Hero.js

```diff
@@ -37,13 +37,25 @@ export default function ContentfulHero({
   }
 
   function list() {
+    headerLinks.forEach(link => {
+      if (link.slug) {
+        link.reference = 'https://www.madetech.com' + link.slug
+        link.display_name = link.name + ' →'
+      } else {
+        link.reference = '#' + link.id
+        link.display_name = '#' + link.name
+      }
+    })
     return (
       <div className="contentful-hero__header-links">
         <ul className="contentful-hero__header-links__ul">
+          <p className="contentful-hero__header-links__title">
+            Jump straight to:
```

Comment:
Wondering if you were planning to have a different text for non-anchor tags, since 'Jump straight to' applies more when it's for anchor tags - if that makes sense? I might have missed something though.

Reply:
You're right! Right now it's just hard-coded as it's one of the things we want to talk about with Rory tomorrow